src/posix/platform/alarm.cpp

```diff
@@ -174,6 +174,7 @@ void otPlatAlarmMicroStartAt(otInstance *aInstance, uint32_t aT0, uint32_t aDt)
     sIsUsRunning = true;
 
 #ifdef __linux__
+#if !OPENTHREAD_POSIX_VIRTUAL_TIME
```

Comment:
nit: should we combine these lines?
```suggestion
#if defined( __linux__) && !OPENTHREAD_POSIX_VIRTUAL_TIME
```